assets/js/theme/category.js

```diff
@@ -18,6 +18,19 @@ export default class Category extends CatalogPage {
             });
         });
 
+        if ($('[data-shop-by-price]').length) {
+            if ($('.navList-action').hasClass('is-active')) {
+                $('a.navList-action.is-active').focus();
+            }
+
+            $('a.navList-action').on('click', () => {
+                $('span.price-filter-message').attr({
+                    role: 'status',
+                    'aria-live': 'assertive',
+                });
+            });
+        }
```

Comment:
Could we move this logic into class method?

Reply:
yes, I've moved setting live region attributes on element in separate method so it's can be reused across the class.